src/System.Runtime.InteropServices.PInvoke/ref/System.Runtime.InteropServices.PInvoke.cs

```diff
@@ -0,0 +1,403 @@
+        public BestFitMappingAttribute(bool BestFitMapping) { }
+        public bool BestFitMapping { get { return default(bool); } }
+    }
+    public enum CallingConvention
+    {
+        Cdecl = 2,
+        StdCall = 3,
+        ThisCall = 4,
+        Winapi = 1,
+    }
+    [System.AttributeUsageAttribute((System.AttributeTargets)(2), Inherited = false)]
+    public sealed partial class DefaultCharSetAttribute : System.Attribute
+    {
+        public DefaultCharSetAttribute(System.Runtime.InteropServices.CharSet charSet) { }
+        public System.Runtime.InteropServices.CharSet CharSet { get { return default(System.Runtime.InteropServices.CharSet); } }
+    }
+    [System.AttributeUsageAttribute((System.AttributeTargets)(2048))]
+    public sealed partial class DefaultParameterValueAttribute : System.Attribute
+    {
+        public DefaultParameterValueAttribute(object value) { }
+        public object Value { get { return default(object); } }
+    }
+    [System.AttributeUsageAttribute((System.AttributeTargets)(64), Inherited = false)]
+    public sealed partial class DllImportAttribute : System.Attribute
+    {
+        public bool BestFitMapping;
+        public System.Runtime.InteropServices.CallingConvention CallingConvention;
+        public System.Runtime.InteropServices.CharSet CharSet;
+        public string EntryPoint;
+        public bool ExactSpelling;
+        public bool PreserveSig;
+        public bool SetLastError;
+        public bool ThrowOnUnmappableChar;
+        public DllImportAttribute(string dllName) { }
+        public string Value { get { return default(string); } }
+    }
+    [System.AttributeUsageAttribute((System.AttributeTargets)(5149), Inherited = false)]
+    public sealed partial class GuidAttribute : System.Attribute
+    {
+        public GuidAttribute(string guid) { }
+        public string Value { get { return default(string); } }
+    }
+    public sealed partial class HandleCollector
+    {
+        public HandleCollector(string name, int initialThreshold) { }
+        public HandleCollector(string name, int initialThreshold, int maximumThreshold) { }
+        public int Count { get { return default(int); } }
+        public int InitialThreshold { get { return default(int); } }
+        public int MaximumThreshold { get { return default(int); } }
+        public string Name { get { return default(string); } }
+        public void Add() { }
+        public void Remove() { }
+    }
+    [System.AttributeUsageAttribute((System.AttributeTargets)(2048), Inherited = false)]
+    public sealed partial class InAttribute : System.Attribute
+    {
+        public InAttribute() { }
+    }
+    public static partial class PInvokeMarshal
+    {
+        public static readonly int SystemDefaultCharSize;
+        public static readonly int SystemMaxDBCSCharSize;
+        [System.Security.SecurityCriticalAttribute]
+        public static System.IntPtr AllocateMemory(int cb) { return default(System.IntPtr); }
```

Comment:
Since it's a new type, can we fix some of the naming guideline issues? e.g. rename cb to sizeInBytes (or something like that).